v2.0/enable-node-map.md

```diff
@@ -4,4 +4,199 @@ summary: Learn how to enable the node map in the Admin UI.
+
+Consider a scenario of a four-node geo-distributed cluster with the following configuration:
+
+|  Node | Region | Datacenter |
+|  ------ | ------ | ------ |
+|  Node1 | us-east-1 | us-east-1a |
+|  Node2 | us-east-1 | us-east-1b |
+|  Node3 | us-west-1 | us-west-1a |
+|  Node4 | eu-west-1 | eu-west-1a |
+
+### Step 1. Ensure the CockroachDB Version is 2.0 or Higher
+
+~~~ shell
+$ cockroach version
+~~~
+
+If not, [upgrade to CockroachDB v2.0](upgrade-cockroach-version.html).
+
+### Step 2. Start (or Restart) the Nodes with the Correct `--locality` Flags
+
+If you have a cluster running without the `--locality` flags set, restart the nodes with the correct `--locality` flags(add link to rolling restart page once it's created).
+
+To start a new cluster with the correct `--locality` flags:
+
+In a new terminal, start Node 1:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach start \
+--insecure \
+--locality=region=us-east-1,datacenter=us-east-1a  \
+--store=node1 \
+--host=<node1 address> \
+--port=26257 \
+--http-port=8080 \
+--join=<node1 address>:26257,<node2 address>:26258,<node3 address>:26259,<node4 address>:26260
+~~~
+
+In a new terminal, start Node 2:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach start \
+--insecure \
+--locality=region=us-east-1,datacenter=us-east-1b \
+--store=node2 \
+--host=<node2 address> \
+--port=26258 \
+--http-port=8081 \
+--join=<node1 address>:26257,<node2 address>:26258,<node3 address>:26259,<node4 address>:26260
+~~~
+
+In a new terminal, start Node 3:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach start \
+--insecure \
+--locality=region=us-west-1,datacenter=us-west-1a \
+--store=node3 \
+--host=<node3 address> \
+--port=26259 \
+--http-port=8082 \
+--join=<node1 address>:26257,<node2 address>:26258,<node3 address>:26259,<node4 address>:26260
+~~~
+
+In a new terminal, start Node 4:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach start \
+--insecure \
+--locality=region=eu-west-1,datacenter=eu-west-1a \
+--store=node4 \
+--host=<node4 address> \
+--port=26260 \
+--http-port=8083 \
+--join=<node1 address>:26257,<node2 address>:26258,<node3 address>:26259,<node4 address>:26260
+~~~
+
+In a new terminal, use the [`cockroach init`](initialize-a-cluster.html) command to perform a one-time initialization of the cluster:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach init --insecure
+~~~
+
+### Step 3. [Set the Enterprise License](enterprise-licensing.html)
+
+### Step 4. Set the Latitudes and Longitudes for the Localities
+
+Launch the built-in SQL client:
+
+{% include copy-clipboard.html %}
+~~~ sql
+$ cockroach sql --insecure --host=<address of any node>
+~~~
+
+Insert the latitudes and longitudes of each region into the `system.locations` table:
```

Comment:
Just had a thought, maybe we should sprinkle the word "approximate" or something when we talk about lat/long, to emphasize that it doesn't need to be perfect?

Reply:
Done.